--- tagging_alchemy_suggest.module	2010-07-14 13:38:49.000000000 +0000
+++ change/tagging_alchemy_suggest.module	2010-10-04 17:59:27.000000000 +0000
@@ -13,6 +13,8 @@
   }
   
   $postContent = $node->body;
+  if( null == $postContent )
+    return;
   $entityResult = $alchemyObj->HTMLGetRankedNamedEntities($postContent, "http://www.alchemyapi.com/tools/alchemytagger/?type=post");
 
   $keywordParams =  new AlchemyAPI_KeywordParams();
@@ -89,4 +91,4 @@
     }
   }
   return $returnKeywordsArr;
-}
\ No newline at end of file
+}
